pkg/project/create.go

```diff
@@ -313,39 +323,35 @@ func renameLegacySettings(pathToLegacySettings string, pathToCwSettings string)
 
 // writeNewCwSettings writes a default .cw-settings file to the given path,
 // dependant on the build type of the project
-func writeNewCwSettings(conID string, pathToCwSettings string, BuildType string) *ProjectError {
-	defaultCwSettings, projErr := getDefaultCwSettings(conID, BuildType)
+func writeNewCwSettings(client utils.HTTPClient, connection *connections.Connection, conURL string, pathToCwSettings string, BuildType string) *ProjectError {
```

Comment:
nit: Think `client` is `httpClient` in every other function that takes it as a parameter. also  `getDefaultCwSettings`

Reply:
Good find, changed